### ISSUE TITLE
Provide package on PyPI

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden-services#67]
## Description

**As Josh I want to be able to**

download the `raiden-services` package from PyPI.

**So that I can**

run the services with tools I know and a minimum of hassle.

## Acceptance Criteria

- [x] Check package build and artifact creation
- [x] Add a changelog
- [x] Make a first release
- [x] Test that the release is installable and working
- [x] Add installation information to the README 
- [x] Add `bumpversion` config, see https://github.com/raiden-network/raiden-services/pull/78#discussion-diff-253821658R29

## Related story / issue
-

8 SP

**Comment by Dominik1999** [comment]
- `pip install raiden-services` works
- `monitoring-service --help` and `pathfinding-service --help` work fine 
- `patfinding-service` gives a warning:
```
(RaidenClient) pirate@pirate-ThinkPad-T480s:~/PycharmProjects/TestRaidenServices$ pathfinding_service 
/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/pathfinding_service/cli.py:3: MonkeyPatchWarning: Monkey-patching ssl after ssl has already been imported may lead to errors, including RecursionError on Python 3.6. It may also silently lead to incorrect behaviour on Python 3.7. Please monkey-patch earlier. See https://github.com/gevent/gevent/issues/1016. Modules that had direct imports (NOT patched): ['urllib3.util.ssl_ (/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/urllib3/util/ssl_.py)', 'urllib3.util (/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/urllib3/util/__init__.py)']. 
```
- `monitoring-service` throws an error 
```
(RaidenClient) pirate@pirate-ThinkPad-T480s:~/PycharmProjects/TestRaidenServices$ monitoring_service 
Traceback (most recent call last):
  File "/usr/lib/python3.7/encodings/hex_codec.py", line 19, in hex_decode
    return (binascii.a2b_hex(input), len(input))
binascii.Error: Non-hexadecimal digit found

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/bin/monitoring_service", line 11, in <module>
    sys.exit(main())
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/monitoring_service/cli.py", line 122, in main
    required_confirmations=confirmations,
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/monitoring_service/service.py", line 43, in __init__
    self.address = private_key_to_address(private_key)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/raiden_libs/utils/signing.py", line 92, in private_key_to_address
    private_key = to_bytes(hexstr=private_key)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/decorators.py", line 75, in wrapper
    return to_wrap(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/conversions.py", line 88, in to_bytes
    return decode_hex(hexstr)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/hexadecimal.py", line 17, in decode_hex
    return codecs.decode(remove_0x_prefix(value), "hex")  # type: ignore
binascii.Error: decoding with 'hex' codec failed (Error: Non-hexadecimal digit found)
(RaidenClient) pirate@pirate-ThinkPad-T480s:~/PycharmProjects/TestRaidenServices$ pathfinding_service
/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/pathfinding_service/cli.py:3: MonkeyPatchWarning: Monkey-patching ssl after ssl has already been imported may lead to errors, including RecursionError on Python 3.6. It may also silently lead to incorrect behaviour on Python 3.7. Please monkey-patch earlier. See https://github.com/gevent/gevent/issues/1016. Modules that had direct imports (NOT patched): ['urllib3.util.ssl_ (/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/urllib3/util/ssl_.py)', 'urllib3.util (/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/urllib3/util/__init__.py)']. 
  monkey.patch_all()  # isort:skip # noqa
02-12 10:52:27 - pathfinding_service.cli - INFO - Starting Raiden Pathfinding Service
02-12 10:52:27 - pathfinding_service.cli - INFO - Using contracts version: 0.3._
02-12 10:52:27 - pathfinding_service.cli - INFO - Starting Web3 client for node at http://localhost:8545
02-12 10:52:27 - pathfinding_service.cli - ERROR - Can not connect to the Ethereum client. Please check that it is running and that your settings are correct.
(RaidenClient) pirate@pirate-ThinkPad-T480s:~/PycharmProjects/TestRaidenServices$ monitoring_service
Traceback (most recent call last):
  File "/usr/lib/python3.7/encodings/hex_codec.py", line 19, in hex_decode
    return (binascii.a2b_hex(input), len(input))
binascii.Error: Non-hexadecimal digit found

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/bin/monitoring_service", line 11, in <module>
    sys.exit(main())
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/monitoring_service/cli.py", line 122, in main
    required_confirmations=confirmations,
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/monitoring_service/service.py", line 43, in __init__
    self.address = private_key_to_address(private_key)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/raiden_libs/utils/signing.py", line 92, in private_key_to_address
    private_key = to_bytes(hexstr=private_key)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/decorators.py", line 75, in wrapper
    return to_wrap(*args, **kwargs)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/conversions.py", line 88, in to_bytes
    return decode_hex(hexstr)
  File "/home/pirate/PycharmProjects/TestRaidenServices/venv/RaidenClient/lib/python3.7/site-packages/eth_utils/hexadecimal.py", line 17, in decode_hex
    return codecs.decode(remove_0x_prefix(value), "hex")  # type: ignore
binascii.Error: decoding with 'hex' codec failed (Error: Non-hexadecimal digit found)
```

**Comment by Dominik1999** [comment]
Bug report for monitoring service issue - #92 (closed)
Warning report for pathfinding service issue- #93

**Comment by Dominik1999** [comment]
closed by #98
